### ISSUE TITLE
iOS: Message input remains after message submission

### ISSUE THREAD

**Issue by rinchen** [Free-Republic-of-Verdis/verdis-communications#2]
After entering and sending a message, sometimes the message will remain in the chat box making you believe the message may not have been sent.
![IMG_90819637B9EB-1](https://user-images.githubusercontent.com/44416/152260974-2b915760-2164-47b9-ad68-7eead6fc06da.jpeg)

**Comment by garv-shah** [comment]
huh, I've never seen this before, I don't even know how it would be possible, let me see if I can reproduce it

**Comment by garv-shah** [comment]
hmm, I can't replicate it, would you be able to send a screen recording, which platforms you can do it on, and the app version?

**Comment by rinchen** [comment]
It's as you see there, and also in our in-system chats. It just starts happening and there is no way to delete it or overwrite it. That's why you see duplicate postings sometimes.

**Comment by garv-shah** [comment]
yeah, I just need to be able to replicate it. would you be able to send the app version and platforms?

**Comment by rinchen** [comment]
iOS v1.0.3

**Comment by garv-shah** [comment]
hmm, I really can't find a way to reproduce this one
I'll leave it open until I can, but I don't think I can fix it if I can't get it to happen

**Comment by garv-shah** [comment]
yeah, I've tried for half an hour to recreate it, I can't really make it happen
I'm open to prs, but it might be caused by some external factor such as ur wifi, I'm not sure
for now, I probably won't be able to fix it without knowing what causes it
